ci-operator/step-registry/mirror-images/by-oc-adm/mirror-images-by-oc-adm-commands.sh

```diff
@@ -20,15 +20,32 @@ if [ ! -f "${SHARED_DIR}/mirror_registry_url" ]; then
 fi
 echo "MIRROR_REGISTRY_HOST: $MIRROR_REGISTRY_HOST"
 
+# ARM clusters need to mirror image from ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}
```

Comment:
I'd use the value of OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE, avoiding specific code for the arm64 cluster architecture.

That is changing all the lines from 23 to 39 to:
```
if [[ -n "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
  source_release_image="${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
else
  source_release_image="${RELEASE_IMAGE_LATEST}"
fi
```